integration-testing/build.gradle

```diff
@@ -2,11 +2,52 @@
  * Copyright 2016-2021 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license.
  */
 
+
+buildscript {
```

Comment:
The script is already close to being 200-lines long.

Could you please migrate it to `.kts` while we are here, so it's easier to maintain?

Reply:
This script contains some non trivial tasks configuration (e.g. [here](https://github.com/Kotlin/kotlinx.coroutines/blob/fbf966e22f9000bc3da262776e82e1221bd08136/integration-testing/build.gradle.kts#L68)) and it turned out to be very time consuming to migrate. It requires some special knowledge, and I failed do it quickly now ☹️. Maybe it's better to do it as a separate PR later?

Reply:
Works for me